cmd/operator-sdk/olmcatalog/cmd.go

```diff
@@ -15,16 +15,70 @@
 package olmcatalog
 
 import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/operator-framework/operator-sdk/internal/pkg/scaffold"
+	"github.com/operator-framework/operator-sdk/internal/pkg/scaffold/input"
+	catalog "github.com/operator-framework/operator-sdk/internal/pkg/scaffold/olm-catalog"
+	"github.com/operator-framework/operator-sdk/internal/util/projutil"
+
+	log "github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 )
 
+var (
+	writeCSVConfigOpt = "write-csv-config"
+
+	writeCSVConfigPath string
+)
+
 func NewCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "olm-catalog <olm-catalog-command>",
+		Use:   "olm-catalog [flags] <olm-catalog-command>",
 		Short: "Invokes a olm-catalog command",
 		Long: `The operator-sdk olm-catalog command invokes a command to perform
 Catalog related actions.`,
+		RunE: olmCatalogFunc,
 	}
 	cmd.AddCommand(newGenCSVCmd())
+
+	i, err := (&catalog.CSVConfig{}).GetInput()
+	if err != nil {
+		log.Fatalf("Error retrieving CSV config path: %v", err)
+	}
```

Comment:
Could not the path be an attribute of the struct and the check be made in the Validate func of it?